pkgs/tools/text/nltk_data/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, newScope, fetchFromGitHub, unzip, stdenv }:
+let
+  base = {
+    version = "unstable-2023-02-02";
+    src = fetchFromGitHub {
+      owner = "nltk";
+      repo = "nltk_data";
+      rev = "5db857e6f7df11eabb5e5665836db9ec8df07e28";
+      sha256 = "sha256-RFywUkesoUL8tbiKLs6Pa+NpoEOq9eu1dP83n5XkNpY=";
+    };
+    nativeBuildInputs = [ unzip ];
+    dontBuild = true;
+    meta = with lib; {
+      description = "NLTK Data";
+      homepage = "https://github.com/nltk/nltk_data";
+      license = licenses.asl20;
+      platforms = platforms.unix;
```

```suggestion
      platforms = platforms.all;
```
It's just data, and I don't think it needs to be limited to unix.